dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblem.java

```diff
@@ -26,17 +28,22 @@
  */
 final class InaccessibleClassProblem extends LinkageProblem {
   private AccessModifier modifier;
+  private ClassSymbol classSymbol;
```

Comment:
why is this stored here and in the superclass? The superclass should be enough.

Reply:
That's a good point. It's not needed. Removed.